Runtime/Scripts/msCoreAPI.cs

```diff
@@ -558,18 +596,41 @@ public struct TextureRecord {
 
     public TextureRecord textureValue {
         get {
+            if (self == IntPtr.Zero) {
+                return default;
+            }
+            
             var ret = default(TextureRecord);
             msMaterialPropCopyData(self, ref ret);
             return ret;
         }
     }
 
+    // Not used yet:
```

Comment:
If we are not using it in this PR, then we should put it in another PR and remove it here.

Reply:
Deleted it now